tests/integration_tests/test_preprocessing.py

```diff
@@ -890,7 +890,7 @@ def test_prompt_template(input_features, expected, model_type, backend, tmpdir,
     model_name = "hf-internal-testing/tiny-random-OPTModel"
     if model_type == MODEL_LLM:
         # For LLMs, specify the prompt at the top level
-        config["model_name"] = model_name
+        config["base_model"] = model_name
```

Comment:
One thought - given the number of places you've had to make the change from `model_name` to `base_model` across all of the files, I'm wondering if it would be a good idea to just make it a constant and use it everywhere so that if we ever make a change to the name in the future we just have to make it in one place

Reply:
it actually already exists as a constant and I forgot to use it, good catch